README.md

```diff
@@ -76,6 +76,33 @@ Where fixes are available within a rule, these can be applied automatically with
 golangci-kube-api-linter run path/to/api/types --fix
 ```
 
+### Golangci-lint Plugin
+
+The Kube API Linter can also be used as a plugin for `golangci-lint`.
+To do this, you will need to install the `golangci-lint` binary and then install the Kube API Linter plugin.
```

Comment:
Do we need to write about that to use plugin mode, golangci-lint must be build with CGO_ENABLED=1?

Reply:
Since the golangci-lint describes that CGO_ENABLED is required in the post.

https://golangci-lint.run/plugins/go-plugins/

> CGO_ENABLED is another requirement.
> This means that golangci-lint needs to be built for whatever machine you intend to run it on (cloning the golangci-lint repository and running a CGO_ENABLED=1 make build should do the trick for your machine).

Reply:
Ah, right. I had forgotten that this comes from golangci-lint and ultimately Go itself, not the linter.

I don't think you need to document it here.

Reply:
It might be helpful to link to the go-plugins documentation though, so I can add that here